### PR TITLE
adding api data in token table

### PR DESCRIPTION
**Pull request by ahmednabil310**

### CODE REVIEW

File: src/components/token/components/section_3.jsx [hunk 1]
```diff
@@ -27,31 +43,34 @@ const SectionThree = () => {
     ],
     []
   )
+
+
+  const {
+    data: holdersData = {
+      bsc: '1,642',
```
Comment (leonprou) [inline src/components/token/components/section_3.jsx:50]: @ahmednabil310 I think you can remove the hardcoded numbers

File: src/components/token/components/section_3.jsx [hunk 2]
```diff
@@ -27,31 +43,34 @@ const SectionThree = () => {
...
+
   const {
-    data: circulatingData = { ethereumBlock: '', bscBlock: '', fuseBlock: '' }
-  } = useFetch('https://bot.fuse.io/api/v1/stats/circulating', {}, [])
-  // TODO
-  // const {
-  //   data: supplyData  = { ethereumBlock: "", bscBlock: "", fuseBlock: "" },
-  // } = useFetch('https://bot.fuse.io/api/v1/stats/supply', {}, []);
+    data: circulatingData = {
+      onBSCNetwork: 18567954,
```
Comment (leonprou) [inline src/components/token/components/section_3.jsx:58]: and here's too

Comment (ahmednabil310) [reply src/components/token/components/section_3.jsx]: done

File: src/components/token/components/section_3.jsx (hunk 1 above)
Comment (ahmednabil310) [reply src/components/token/components/section_3.jsx]: done, btw they were a backup default values

**Pull request merged**